shell/browser/native_browser_view_views.cc

```diff
@@ -92,9 +92,35 @@ void NativeBrowserViewViews::ResetAutoResizeProportions() {
 }
 
 void NativeBrowserViewViews::SetBounds(const gfx::Rect& bounds) {
+  // Previous implementation
+  /*
   auto* view = GetInspectableWebContentsView()->GetView();
   view->SetBoundsRect(bounds);
   ResetAutoResizeProportions();
+  */
+
+  /*
+  This implementation is attempting to ignore setting bounds
+  if the view has been destroyed, (ie invalid).
+  A different check is most likely needed as the destroyed
```

Comment:
Very correct here, the best thing you can probably do is make `NativeBrowserViewViews` a `WebContentsObserver` and manually set `inspectable_web_contents_ = nullptr` when the webcontents is destroyed.

You can look at `BrowserView::WebContentsDestroyed` as an example of this pattern

Reply:
For the above, you can take a look [here](https://github.com/electron/electron/blob/3ac3fbdbfb4af9d7b35b02b0c3b4b8307c614c08/shell/browser/web_contents_zoom_controller.h#L20), and [here](https://github.com/electron/electron/blob/3ac3fbdbfb4af9d7b35b02b0c3b4b8307c614c08/shell/browser/web_contents_zoom_controller.h#L85-L89), though in this case we'll only want to override `WebContentsDestroyed()`